README.md

```diff
@@ -11,6 +11,14 @@ The unipipe cli ships as a single, fully self-contained binary. This allows the
 cli to be easily run in any CI/CD systems that supports a bash script job like
 Jenkins, Azure DevOps, GitLab or GitHub actions.
 
+The Unipipe CLI supercharges your GitOps OSB service pipelines. If you want to
+use our open source
```

Comment:
To me the text sounds a bit like using unipipe osb is "optional" - it's kind of a key prerequisite right? I'd find sth along the lines of "unipipe-cli enables you to quickly deploy and build services on top of unipipe-service-broker" more intuitive